### DISCUSSION TITLE
Feature request: Autostart, system tray and autoplay

### DISCUSSION THREAD

**Discussion by basxx2** [tom-englert/RegionToShare#2] [Ideas]
Hello, 
First of all, great to see this small app around. I was fighting with the Teams sharing issue on high-resolution monitors for quite a while now and I believe this goes into a good direction until Teams supports sharing custom sized windows natively. 
I have the following three suggestions to be added as a setting to this app: 
- Flag to start the app upon Windows start
- Start the app to the system tray rather than as a “normal program windows”
- Option to toggle autoplay (and maybe remove or hide the bezels) so that it can run all the time in the background and you would just need to share the window in Teams and drag your content to the area 

Thanks and keep up the great work
Basti

**Comment by tom-englert** [comment]
> Start the app to the system tray rather than as a “normal program windows”

Will it then still be discoverable by Teams? I can't share any window that only has a system tray icon.

> hide the bezels

How would you then know the area where to put your content to share?

Some background why it allows to toggle between inactive and active mode:
- When inactive, it uses nearly no processor resources, but when active it does
- When inactive, the window is on top and easy to find in Teams or WebEx. When active it's the bottom most window, and you have to always scroll down the list of windows to find it. You can try this by first making it active, and then start sharing.

**Reply by basxx2** [reply]
Thanks for the quick reply. 
I feel a little stupid now because indeed all of your comments make sense :-) 
I guess I just did not realize all the why’s and how’s and it seems it’s all well thought of. 

Again, thanks for the great work.

**Comment by tarltonferrin** [comment]
Nice App, Thanks! This will definitely help my day-to-day now that I've adopted a larger work monitor. The main drawback to the monitor was screen sharing and I think this will help a lot. Just did an initial test and it works and am looking forward to using it more in meetings.
